Makefile

```diff
@@ -27,7 +27,7 @@ test:
 	npm test
 
 test-e2e:
-	npx testcafe firefox test/e2e/ -s takeOnFails=true
+	npx testcafe chrome test/e2e/ -s takeOnFails=true
```

Comment:
Why?

Although, I guess we should run both? (but not on every run... can wait for later PRs)

Reply:
Just wanted to harmonize with CI, if in future we add both in CI we can add both in local, else it would mislead development in local (extra errors maybe?). And we need to spend time as to by local tests pass and CI fails or vice-versa.